src/views/cart.hbs

```diff
@@ -0,0 +1,38 @@
+<main class="cart">
+    <section>
+        <div class="cart-header">
+            <div class="cart-header__text">My Cart {{cart.count}} items</div>
+        </div>
+        {{#each cart.items}}
+        <div class="cart-item" data-id="{{product.id}}">
+            <img src="{{product.imageURL}}" alt="{{product.name}}" class="cart-item__logo" />
+            <div class="cart-item-detail">
+                <h2 class="cart-item-detail__heading">{{product.name}}</h2>
+                <div class="cart-item-detail__info">
+                    <div class="cart-item-detail__info--count">
+                        <a class="cart-item-detail__info--remove" data-id="{{product.id}}">&#45;</a>
```

Comment:
is there a specific naming convention you are following for class names ?

Reply:
done changes